shipment_advice_planner_toursolver/models/toursolver_backend.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class TourSolverBackend(models.Model):
+    _name = "toursolver.backend"
```

Comment:
A companion addon will be required to managed part of these informations (url, api_key) from server_environment